src/main/java/me/amarpandey/config/SchedulerConfig.java

```diff
@@ -17,6 +19,6 @@
 
 	@Scheduled(fixedDelay = 3000)
 	public void sendAdhocMessages() {
-		template.convertAndSend("/topic/stats", count);
+    template.convertAndSend(Constants.TOPIC_STATS, count);
```

Comment:
make `TOPIC_STATS` a static import

Reply:
Its defined in Contants.java
Why do you want to specifically import in each file?

Reply:
there is no need to import `Constants` when we need only `TOPIC_STATS`.

Inshort, use 
```java
import static me.amarpandey.utils.Constants.TOPIC_STATS;
```
Instead
```java
import me.amarpandey.utils.Constants;
```